loadtest/src/tests/events-verify.ts

```diff
@@ -179,32 +215,170 @@ export default async function () {
+              `Invalid internal latency calculation for event ${finalEventId}: ${intLatency}`
+            );
+          }
+        }
+
+        // After all latency calculations, log a summary for this event
+        if (sentTimestamp) {
+          // Calculate the expected sum to verify our metrics add up
+          const receiveLatencyVal =
+            outpostReceivedTimestamp && sentTimestamp
+              ? outpostReceivedTimestamp - sentTimestamp
+              : 0;
+          const internalLatencyVal =
+            deliveryStartTimestampMs && outpostReceivedTimestamp
+              ? Math.max(0, deliveryStartTimestampMs - outpostReceivedTimestamp)
+              : 0;
+          const deliveryLatencyVal =
+            receivedTimestampMs && deliveryStartTimestampMs
+              ? receivedTimestampMs - deliveryStartTimestampMs
+              : 0;
+
+          const totalCalculatedLatency =
+            receiveLatencyVal + internalLatencyVal + deliveryLatencyVal;
+
+          // console.log(`Event ${finalEventId} latency summary (milliseconds):
+          //   Sent: ${
+          //     sentTimestamp ? new Date(sentTimestamp).toISOString() : "missing"
+          //   }
+          //   Received by Outpost: ${
+          //     outpostReceivedTimestamp
+          //       ? new Date(outpostReceivedTimestamp).toISOString()
+          //       : "missing"
+          //   }
+          //   Delivery started: ${
+          //     deliveryStartTimestampMs
+          //       ? new Date(deliveryStartTimestampMs).toISOString()
+          //       : "missing"
+          //   }
+          //   Received by endpoint: ${
+          //     receivedTimestampMs
+          //       ? new Date(receivedTimestampMs).toISOString()
+          //       : "missing"
+          //   }
+          //   End-to-end: ${
+          //     receivedTimestampMs && sentTimestamp
+          //       ? receivedTimestampMs - sentTimestamp
+          //       : "N/A"
+          //   }
+          //   Receive: ${
+          //     outpostReceivedTimestamp && sentTimestamp
+          //       ? outpostReceivedTimestamp - sentTimestamp
+          //       : "N/A"
+          //   }
+          //   Internal: ${
+          //     deliveryStartTimestampMs && outpostReceivedTimestamp
+          //       ? Math.max(
+          //           0,
+          //           deliveryStartTimestampMs - outpostReceivedTimestamp
+          //         )
+          //       : "N/A"
+          //   }
+          //   Delivery: ${
+          //     receivedTimestampMs && deliveryStartTimestampMs
+          //       ? receivedTimestampMs - deliveryStartTimestampMs
+          //       : "N/A"
+          //   }
+          //   Total calculated: ${totalCalculatedLatency}
+          //   Diff from E2E: ${
+          //     receivedTimestampMs && sentTimestamp
+          //       ? receivedTimestampMs - sentTimestamp - totalCalculatedLatency
+          //       : "N/A"
+          //   }`);
```

Comment:
@alexluong - do you want this commented out?

Reply:
Yes, it's just a debug log line. It's helpful to have to confirm the timestamps are correct, so I still use it sometimes when developing (both outpost & loadtest), but when running the test, you don't these logs

Reply:
Ah, ok. I think some more of the code that's only used in the logging could also be commented out to avoid confusion.